routes/create.js

```diff
@@ -4,42 +4,60 @@ var validator = require("email-validator");
 var User = require('../models/user');
 
 /*Renders different page based on if user is logged in or not*/
-router.get('/', function(req, res){
-	if (req.user) {
-		req.user.hyphenedCode = insertHyphen(req.user.code);
-		res.render('create/create');
-	} else {
-		res.render('create/guestcreate');
-	}
+router.get('/', function (req, res) {
```

Comment:
why not split this into different functions. One for user logged in, another not verified.